Added transactions amounts for ERM cost query and prepared derived tables for that

adoerrer opened the pull request:
will fix #375 

This can be tested against 'folio_snapshot_20210218_invoice' but  you need to change the joined tables to `local.axeldoerrer_invoice_adjustments_ext `and `local.axeldoerrer_finance_transaction_invoices`. And you have remove the preset filters to get results.

lib-eric commented on sql/report_queries/erm/costs/erm_inventory_cost_trans.sql:
```diff
@@ -0,0 +1,180 @@
+        -- filters on purchase order line level
+        ''::VARCHAR AS po_line_order_format, -- Enter your po line format eg. 'Electronic Resource', 'Physical Resource', 'P/E Mix' etc.
+        -- filters on instance level
+        ''::VARCHAR AS instance_subject, -- Enter your  eg.  etc.
```
Line 38, 39, 41:
Should there be an example in the comments for filters?

adoerrer replied:
Have added them now.

lib-eric reviewed (commented):
Is 'erm_inventory_cost.sql ' and 'erm_inventory_cost_trans.sql' supposed to be exactly the same? This might be your commit about "deleted old query and renamed new one" but it looks like they are both still there.

Able to run derived tables and queries to get results after making mentioned changes to use local and remove filters.

Someone else may want to take a look at the numbers that better understands invoicing. The numbers aren't quite making sense to me.

adoerrer commented:
> 
> 
> Is 'erm_inventory_cost.sql ' and 'erm_inventory_cost_trans.sql' supposed to be exactly the same? This might be your commit about "deleted old query and renamed new one" but it looks like they are both still there.
> 
> Able to run derived tables and queries to get results after making mentioned changes to use local and remove filters.
> 
> Someone else may want to take a look at the numbers that better understands invoicing. The numbers aren't quite making sense to me.

Thanks, in my local repo it appeared to be deleted. I now corrected that.

adoerrer commented on sql/report_queries/erm/costs/erm_inventory_cost_trans.sql:
```diff
@@ -0,0 +1,180 @@
+    invl_adjustments.adjustment_description AS "invl_adjustment_description",
+    invl_adjustments.adjustment_prorate AS "invl_adjustment_prorate",
+    invl_adjustments.adjustment_relationtototal AS "invl_adjustment_relationtototal",
+    invl_adjustments.adjustment_value AS "invl_adjustment_value",
```
Thanks that's a good point. Although all totals and values are system calculated to prevent confusing we should add adjustment_type.

lib-eric reviewed (approved):
Looks good to me!
